package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "umbra",
+  "version": "0.0.1",
+  "description": "Send and receive stealth payments with the Umbra protocol",
+  "author": "mds <17163988+mds1@users.noreply.github.com>",
```

@apbendi Should we update the author in all `package.json` files to something like this? 

```suggestion
  "author": "ScopeLift <contact@scopelift.co>",
```